Gemfile

```diff
@@ -25,7 +25,7 @@ gem "carrierwave-bombshelter", "~> 0.2" # Protect your carrierwave from image bo
 gem "cloudinary", "~> 1.18" # Client library for easily using the Cloudinary service
 gem "counter_culture", "~> 2.7" # counter_culture provides turbo-charged counter caches that are kept up-to-date
 gem "ddtrace", "~> 0.45.0" # ddtrace is Datadog’s tracing client for Ruby.
-gem "devise", "~> 4.7" # Flexible authentication solution for Rails
+gem "devise", github: "heartcombo/devise", ref: "0cd72a5" # Flexible authentication solution for Rails
```

Comment:
we could add a note with https://github.com/forem/forem/pull/12715#issuecomment-781026686 explaining why we decide to merge with the unreleased devise

Reply:
👍 , I'll add that tomorrow.